The Nobel Prize has been among the most prestigious international awards since 1901. Each year, awards are bestowed in chemistry, literature, physics, physiology or medicine, economics, and peace. In addition to the honor, prestige, and substantial prize money, the recipient also gets a gold medal with an image of Alfred Nobel (1833 - 1896), who established the prize.

![](Nobel_Prize.png)

The Nobel Foundation has made a dataset available of all prize winners from the outset of the awards from 1901 to 2023. The dataset used in this project is from the Nobel Prize API and is available in the `nobel.csv` file in the `data` folder.

In this project, you'll get a chance to explore and answer several questions related to this prizewinning data. And we encourage you then to explore further questions that you're interested in!

In [239]:
# Loading in required libraries
import pandas as pd
import seaborn as sns
import numpy as np

# Start coding here!
nobel_df = pd.read_csv("data/nobel.csv")
nobel_df.shape
pd.set_option('display.max_columns', None)
print(nobel_df.head())

   year    category                                           prize  \
0  1901   Chemistry               The Nobel Prize in Chemistry 1901   
1  1901  Literature              The Nobel Prize in Literature 1901   
2  1901    Medicine  The Nobel Prize in Physiology or Medicine 1901   
3  1901       Peace                      The Nobel Peace Prize 1901   
4  1901       Peace                      The Nobel Peace Prize 1901   

                                          motivation prize_share  laureate_id  \
0  "in recognition of the extraordinary services ...         1/1          160   
1  "in special recognition of his poetic composit...         1/1          569   
2  "for his work on serum therapy, especially its...         1/1          293   
3                                                NaN         1/2          462   
4                                                NaN         1/2          463   

  laureate_type                     full_name  birth_date         birth_city  \
0    I

# **The Most Commonly Awarded Gender and Birth Country**

In [240]:
top_gender=nobel_df['sex'].value_counts().idxmax() 
gender_awarded=nobel_df['sex'].value_counts()
print(f"Total Number of Awarded Gender {gender_awarded}")
print(f"The Most Awarded Gender is: {top_gender}")
top_country=nobel_df['birth_country'].value_counts().idxmax()
birth_countries=nobel_df['birth_country'].value_counts()
print(f"Birth Countries of Award Ricipents are {birth_countries}")
print(f"The Most Commonly Awarded Birth Country is {top_country}")

Total Number of Awarded Gender Male      905
Female     65
Name: sex, dtype: int64
The Most Awarded Gender is: Male
Birth Countries of Award Ricipents are United States of America                        291
United Kingdom                                   91
Germany                                          67
France                                           58
Sweden                                           30
                                               ... 
Austria-Hungary (Ukraine)                         1
Ottoman Empire (Turkey)                           1
Russian Empire (Azerbaijan)                       1
Bosnia (Bosnia and Herzegovina)                   1
French protectorate of Tunisia (now Tunisia)      1
Name: birth_country, Length: 129, dtype: int64
The Most Commonly Awarded Birth Country is United States of America


In [241]:
nobel_df['decade'] = (nobel_df['year'] // 10) * 10
nobel_df['us_born'] = nobel_df['birth_country'].str.contains('United States|USA|US', case=False, na=False)
decade_stats = nobel_df.groupby('decade').agg(
    total_winners=('laureate_id', 'count'),
    us_winners=('us_born', 'sum')
)
decade_stats['us_ratio'] = decade_stats['us_winners'] / decade_stats['total_winners']
max_decade_usa = int(decade_stats['us_ratio'].idxmax()) 
print(f"The decade with the highest ratio of US-born Nobel Prize winners is the {max_decade_usa}s")

The decade with the highest ratio of US-born Nobel Prize winners is the 2000s


In [242]:
female_laureates = nobel_df[nobel_df['sex'] == "Female"]
female_by_decade_category = female_laureates.groupby(['decade', 'category']).size()
total_by_decade_category = nobel_df.groupby(['decade', 'category']).size()
femaleratio = (female_by_decade_category / total_by_decade_category).fillna(0)
max_female_decade_category = femaleratio.sort_values(ascending=False).idxmax()
max_female_dict = {max_female_decade_category[0]: max_female_decade_category[1]}
print(f"Decade and Nobel Prize category with the highest proportion of female laureates: {max_female_dict}")

Decade and Nobel Prize category with the highest proportion of female laureates: {2020: 'Literature'}


In [243]:
first_woman_name = nobel_df[nobel_df['sex'] == "Female"].sort_values('year').iloc[0]['full_name']
print(first_woman_name)
first_woman_category = nobel_df[nobel_df['sex'] == "Female"].sort_values('year').iloc[0]['category']
print(first_woman_category)

Marie Curie, née Sklodowska
Physics


In [244]:
most_winner = nobel_df['full_name'].value_counts()
repeat_list = most_winner[most_winner > 1].index.tolist()
print(repeat_list)

['Comité international de la Croix Rouge (International Committee of the Red Cross)', 'Linus Carl Pauling', 'John Bardeen', 'Frederick Sanger', 'Marie Curie, née Sklodowska', 'Office of the United Nations High Commissioner for Refugees (UNHCR)']


From the analysis, it is found that: 

- Male is the most commonly awarded gender with 905 winners while Female is 65 winners. 
- United States of America is the most commonly awarded country
- The highest ratio of female winners is in 2020s (decade)
- The highest ratio of US-born winners is in 2000s (decade)
- Marie Curie, née Sklodowska is the first female winner and the category is physics in 1903
- The list of individuals or organization which have won more than one Nobel Prize are: 
 1. Comité international de la Croix Rouge (International Committee of the Red Cross)
 2. Linus Carl Pauling
 3. John Bardeen
 4. Frederick Sanger
 5. Marie Curie, née Sklodowska
 6. Office of the United Nations High Commissioner for Refugees (UNHCR)
